### PR TITLE
[All] Rename Accessibility -> AutomationProperties

### PR DESCRIPTION
**Pull request by samhouts**
### Description of Change ###

Renamed `Accessibility` to `AutomationProperties`. ~~Based from #911. ~~

#889 and #904 will need to be rebased if this is merged first.

UPDATE: idem for #913, #914

### Bugs Fixed ###

None

### API Changes ###

Same properties exist from #713, but the namespace is now `AutomationProperties`.

### Behavioral Changes ###

None

### PR Checklist ###

- [x] Has tests (if omitted, state reason in description)
- [x] Rebased on top of master at time of PR
- [x] Changes adhere to coding standard
- [x] Consolidate commits as makes sense

### CODE REVIEW

File: Xamarin.Forms.Core/AutomationProperties.cs [hunk 1]
```diff
@@ -1,19 +1,19 @@
 ﻿﻿
 namespace Xamarin.Forms
 {
-	public class Accessibility
+	public class AutomationProperties
```
Comment (StephaneDelcroix) [inline Xamarin.Forms.Core/AutomationProperties.cs]: when was this released first ? isn't this a breaking change ?

**Comment by StephaneDelcroix** [comment]
Rebased this branch. In case I made a mess, you van restore the old HEAD with 
```
git reset --hard a117-rename cc8320c2c55de350050115142844f0ee18fac583
git push -f
```

File: Xamarin.Forms.Core/AutomationProperties.cs (hunk 1 above)
Comment (DanRigby) [inline Xamarin.Forms.Core/AutomationProperties.cs]: @StephaneDelcroix 2.3.5-pre1 is when this class first appeared in a NuGet release.

File: Xamarin.Forms.Core/Xamarin.Forms.Core.csproj [hunk 2]
```diff
@@ -477,7 +477,5 @@
   </PropertyGroup>
   <ItemGroup />
   <ItemGroup />
-  <ItemGroup>
```
Comment (rmarinho) [inline Xamarin.Forms.Core/Xamarin.Forms.Core.csproj]: is this not needed ?

File: Xamarin.Forms.Core/Xamarin.Forms.Core.csproj [hunk 3]
```diff
@@ -477,7 +477,5 @@
...
-    <Folder Include="Xaml\" />
```
Comment (rmarinho) [inline Xamarin.Forms.Core/Xamarin.Forms.Core.csproj]: Sorry is this not needed ?

**Pull request merged**